client/components/account-balances/balance-tooltip.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import { sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from 'components/tooltip';
+import {
+	documentationUrls,
+	fundLabelStrings,
+	fundTooltipStrings,
+	learnMoreString,
+} from './strings';
+import { balanceType } from './balance-block';
+
+type BalanceTooltipProps = {
+	type: balanceType;
+	delayDays?: number;
+	isNegativeBalance?: boolean;
+	className?: string;
+};
+
+const BalanceTooltip: React.FC< BalanceTooltipProps > = ( {
+	type,
+	delayDays,
+	isNegativeBalance,
+	className,
+} ) => {
+	let content = null;
+	const tooltipButtonLabel = `${ fundLabelStrings[ type ] } tooltip`;
+
+	if ( type === 'available' ) {
```

Comment:
Creating an `if` statement for each type is verbose, but I kept it like this for clarity.